.gitignore

```diff
@@ -12,4 +13,10 @@
 .venv
 __pycache__
 /venv/*
-.python-version
\ No newline at end of file
+.python-version
+
+# Envrc
+.envrc.chamber
+.envrc.local*
+exp_*.pem
+stg_*.pem
```

Comment:
why is this in here?
did you use a staging cert?

Reply:
no no, just subconsciously taking it into account. I'll remove it